huaweicloud/services/acceptance/css/resource_huaweicloud_css_cluster_test.go

```diff
@@ -147,6 +150,8 @@ func testAccCssCluster_basic(rName string, nodeNum int, keepDays int, tag string
 resource "huaweicloud_css_cluster" "test" {
   name            = "%s"
   engine_version  = "7.10.2"
```

Comment:
```suggestion
  name           = "%s"
  engine_version = "7.10.2"
```

please fix the format issue

Reply:
done